src/containers/notification/NotificationBadgeWithPopover.ts

```diff
@@ -0,0 +1,20 @@
+import { connect, Dispatch, MapDispatchToProps } from 'react-redux';
+
+import { bindActionCreators } from 'redux';
+import { acknowledgeNotifications } from '../../actions';
+import NotificationBadgeWithPopover, {
+  DispatchProps
+} from '../../components/notification/NotificationBadgeWithPopover';
+
+const mapDispatchToProps: MapDispatchToProps<DispatchProps, {}> = (dispatch: Dispatch<any>) =>
+  bindActionCreators(
+    {
+      handleAcknowledgeNotifications: acknowledgeNotifications
+    },
+    dispatch
+  );
+
+export default connect(
+  null,
```

Comment:
Do you need the null here?

Reply:
Yes. Without the null will invoke a different overloaded function.